### ISSUE TITLE
Reorder New Resource Fossil Collection Event list 

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#650]
- [x] Start with Fossil Abundance
- [x] Create Header "Collection Details"
- [x] Under this, include Subheader Collection Year
- [x] Move Collection End Year (Historical data only) into Historical Data
- [x] include Subheader Collectors
- [x] include Subheader Collection Event Location - this needs to go after Field Notes because these attributes are on a separate card.
- [x] include Subheader Collection Notes
- [x] Keep Header Collection Event Significance, but move up in the list
- [x] Next have Samples Collected
- [x] End list with Historical Data

**Comment by bferguso** [comment]
@LeannePyle - I'm a bit confused here. Are you saying we need to move Collection Year, Collectors, Collection Event Location, Collection Notes under a single card called "Collection Details?"?

**Comment by LeannePyle** [comment]
I'm fine with how it looks currently, but thought we discussed nesting all the "details" such as year, Collectors, the location and notes as subheaders under "Collection Details" - maybe I mis-heard the suggestion

**Comment by bferguso** [comment]
I very well may have.. it's been a while. I think it was about trying to get rid of the nodes with just one value in each. Thanks for the reminder.
The only thing is that Notes will need to go before Location because Location should go on its own card and separate cards need to be at the end.

**Comment by bferguso** [comment]
There are ~ 450 rows with collect event start date lists (eg 1998, 1999 or 1961, 1981). We are no longer supporting date ranges for start-end date. Is it OK to put the second date in the `Collection End Year` which is now in the Historical Data card? If so, does this conversion need to be tracked anywhere?

**Comment by LeannePyle** [comment]
Yes, just retain the first year and moving End Year into Historical is fine. I do not see a need to track this.

**Comment by bferguso** [comment]
Automated cleanup
